custom-collections.go

```diff
@@ -0,0 +1,58 @@
+package shopify
+
+import "time"
+
+// CustomCollections is a list of CustomCollections.
+type CustomCollections []CustomCollection
+
+// CustomCollection is a grouping of products that a merchant can create to make their store easier to browse.
+type CustomCollection struct {
```

Comment:
this struct needs to have all of the methods on it for the Collection interface otherwise it cant be used as a Collection